website/theme.config.tsx

```diff
@@ -0,0 +1,80 @@
+import { DocsThemeConfig } from "nextra-theme-docs";
+import { Footer } from "./components/Footer";
+import CloudQueryLogo from "./components/logos/CloudQuery";
+
+const theme: DocsThemeConfig = {
+  github: "https://github.com/cloudquery/cloudquery",
+  projectLink: "https://github.com/cloudquery/cloudquery",
+  docsRepositoryBase:
+    "https://github.com/cloudquery/cloudquery/blob/main/docs/pages",
+  titleSuffix: " | cloudquery",
+  search: true,
+  floatTOC: true,
+  font: false,
+  projectChatLink: "https://cloudquery.io/discord",
+  feedbackLink: "Question? Give us feedback →",
+  logo: function LogoActual() {
+    return (
+      <>
+        <CloudQueryLogo height={32} />
+        <span className="sr-only">CloudQuery</span>
+      </>
+    );
+  },
+  head: (
+    <>
```

Comment:
Small issue here - we can't use router so we can add `og:url` and `description` here 

https://github.com/shuding/nextra/issues/679